benchmarks/transformer/score_mod.py

```diff
@@ -198,15 +202,7 @@ def relative_bias(score, b, h, m, n):
 
     def head_bias(score, b, h, m, n):
         return score + 2 * h
-
```

Comment:
Anti pathogical?

Reply:
yeah, it makes the benchmarks take too long lol.